PowerShellGet/PSModule.psm1

```diff
@@ -2873,18 +3005,65 @@ function Publish-Script
+                        if (-not $Force -and ($galleryScriptPrerelease -gt $toPublishScriptPrerelease))
+                        {
+                            $message = $LocalizedData.ScriptVersionShouldBeGreaterThanGalleryVersion -f ($scriptName,
+                                                                                                     $PSScriptInfo.Version,
+                                                                                                     $currentPSGetItemInfo.Version,
+                                                                                                     $currentPSGetItemInfo.RepositorySourceLocation)
+                            ThrowError -ExceptionName "System.InvalidOperationException" `
+                                    -ExceptionMessage $message `
+                                    -ErrorId "ScriptVersionShouldBeGreaterThanGalleryVersion" `
+                                    -CallerPSCmdlet $PSCmdlet `
+                                    -ErrorCategory InvalidOperation
+                        }
+
+                        # elseif ($galleryScriptPrerelease -lt $toPublishScriptPrerelease) --> allow publish
+                    }
+                }
+                elseif (-not $Force -and ($galleryScriptVer -gt $toPublishScriptVer))
                 {
                     $message = $LocalizedData.ScriptVersionShouldBeGreaterThanGalleryVersion -f ($scriptName,
-                                                                                                 $PSScriptInfo.Version,
-                                                                                                 $currentPSGetItemInfo.Version,
-                                                                                                 $currentPSGetItemInfo.RepositorySourceLocation)
+                                                                                                     $PSScriptInfo.Version,
+                                                                                                     $currentPSGetItemInfo.Version,
+                                                                                                     $currentPSGetItemInfo.RepositorySourceLocation)
                     ThrowError -ExceptionName "System.InvalidOperationException" `
-                               -ExceptionMessage $message `
-                               -ErrorId "ScriptVersionShouldBeGreaterThanGalleryVersion" `
-                               -CallerPSCmdlet $PSCmdlet `
-                               -ErrorCategory InvalidOperation
+                            -ExceptionMessage $message `
+                            -ErrorId "ScriptVersionShouldBeGreaterThanGalleryVersion" `
+                            -CallerPSCmdlet $PSCmdlet `
+                            -ErrorCategory InvalidOperation
                 }
+
+                # else ($galleryScriptVer -lt $toPublishScriptVer) --> allow publish

```

Comment:
># else ($galleryScriptVer -lt $toPublishScriptVer) --> allow publish [](start = 16, length = 68)

This comment and above look like commented code, consider adding the descriptive comments.